app/models/company.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: companies
+#
+#  id          :bigint           not null, primary key
+#  avatar      :string
+#  description :text
+#  domain      :string
+#  name        :string           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  account_id  :bigint           not null
+#
+# Indexes
+#
+#  index_companies_on_account_id             (account_id)
+#  index_companies_on_domain_and_account_id  (domain,account_id)
+#  index_companies_on_name_and_account_id    (name,account_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (account_id => accounts.id)
+#
```

Comment:
We do not use foreign keys. Similar thoughts as https://github.com/github/gh-ost/issues/331#issuecomment-266027731